indicator_nara/01_inleiding.Rmd

```diff
@@ -1,71 +1,94 @@
 # Inleiding {#h1:inleiding}
 Driekwart van de voor Vlaanderen en Europa meest waardevolle natuurtypes, steunen voor hun voortbestaan op grondwater. Grondwaterafhankelijke vegetaties komen voor in en op de rand van kwelgebieden, beekvalleien, vijvers en poelen. 
-De vochtige gebieden vertonen vaak veel gradiënten en herbergen daarom een hoge biodiversiteit (De Becker, 2020). 
-Als het grondwater door verdroging van samenstelling verandert, of systematisch buiten het bereik van de plantenwortels en de bovengrondse delen van waterplanten valt, krijgen typische soorten het moeilijk. 
+De vochtige gebieden vertonen vaak veel gradiënten en herbergen daarom een hoge biodiversiteit [@Becker2020]. 
+
+De jongste jaren kent Vlaanderen regelmatig langdurige periodes van droogte.
+Droogte is op zich een natuurlijk verschijnsel. 
+Als gevolg van een tekort aan neerslag is de vraag van planten naar (grond)water hoger dan het aanbod, waardoor het (grond)waterpeil zal dalen.
+Na zonneschijn komt er regen: droogte heeft een tijdelijk karakter.
+
+Men mag het tijdelijk karakter echter niet onderschatten.
+Om de droogte in beeld te brengen volstaat immers niet alleen een beeld te geven van het neerslagtekort. 
+Hoewel het neerslagtekort een drijvende kracht is voor het optreden van grondwaterdalingen, zijn er andere factoren die meespelen hoe dat neerslagtekort zich effectief in peilverlagingen zal vertalen. 
```

Comment:
Hier introduceer je voor het eerst je focus op grondwater, maar doet dit m.i. veeleer impliciet. Ik zou hoger een zin toevoegen dat het doel van deze monitoring en dus van de indicator is om (acuut) zicht te krijgen op grondwaterstandsdalingen, gezien het belang hiervan voor grondwaterafhankelijke natuur. Momenteel staat dit nog niet vermeld. Het is volgens mij het argument waarom je niet alle facetten van verdroging, alsook niet-grondwaterafhankelijke natuur, onderzoekt.

Reply:
Ik heb de inleiding in die zin herschikt.

Reply:
Prima :smile_cat:  - hoewel ik kan het niet zien (staat nog lokaal wellicht) :yum: